modules/doc/content/newsletter/2023/2023_07.md

```diff
@@ -6,8 +6,8 @@
 
 MOOSE now supports p-refinement, e.g. refining or coarsening the order of
 the finite element basis functions. In order to use p-refinement, you
-can simply set `switch_h_to_p_refinement=true` in the `Adaptivity` block.
-As suggested by the parameter name, only one of h- or p-refinement may be used
+can simply set `adaptivity_type=p` in the `Adaptivity` block.
+As suggested by the parameter name, only one of h- or p- or hp-refinement may be used
```

Comment:
Oh, heck no.  This is the newsletter, not the manual.  We should be rewriting user documentation, not history.

Reply:
Hahaha will fix